public/app/plugins/datasource/dashboard/datasource.ts

```diff
@@ -27,13 +27,7 @@ export class DashboardDatasource extends DataSourceApi<DashboardQuery> {
 
   query(options: DataQueryRequest<DashboardQuery>): Observable<DataQueryResponse> {
     const sceneScopedVar: ScopedVar | undefined = options.scopedVars?.__sceneObject;
-    let scene: SceneObject | undefined;
-
-    if (!(sceneScopedVar instanceof SafeSerializableSceneObject)) {
-      throw new Error('Scene object from scopedVars is not safe serializable.');
-    }
-
-    scene = sceneScopedVar.valueOf();
+    let scene: SceneObject | undefined = sceneScopedVar ? (sceneScopedVar.value.valueOf() as SceneObject) : undefined;
```

Comment:
@ivanortegaalba - the problem in the backport was here, the lack of `.value.`

Reply:
This already assumed this is SafeSerializableSceneObject, while it can be it or just plain scoped var. But SafeSerializableSceneObject implements ScopedVar interface